pkg/distributor/ha_tracker.go

```diff
@@ -94,7 +94,7 @@ type HATrackerConfig struct {
 	// more than this duration
 	FailoverTimeout time.Duration `yaml:"ha_tracker_failover_timeout"`
 
-	KVStore kv.Config
+	KVStore kv.Config `yaml:"kvstore" doc:"description=Backend storage to use for the ring. Please be aware that memberlist is not supported by the HA tracker since Gossip propagation is too slow for HA purposes."`
```

Comment:
No need to capitalize gossip.

```suggestion
	KVStore kv.Config `yaml:"kvstore" doc:"description=Backend storage to use for the ring. Please be aware that memberlist is not supported by the HA tracker since gossip propagation is too slow for HA purposes."`
```